### ISSUE TITLE
Create & Implement "Site" Styled Components

### ISSUE THREAD

**Issue by thecodeclimber** [thecodeclimber/bluejay-redparts#5]
# Description
Convert _site.scss styles into Styled Components.

# Explanation

## New Folders
I have created some new folders and files:

* `src/styled-components`
* `src/styled-components/common`
* `src/styled-components/common/Site.js`

This structure is parallel to the `scss` folder.

## Create Styled Components
I want to transform each class in `_site.scss` into styled components.  For example:

```
.site --> const Site = styled.div``
.site__container --> const SiteContainer = styled.div``
.site__header --> const SiteHeader = styled.header``
...
```

NOTE:  You can find out which html tags to use with the styled components from the `Layout.tsx` file (see resources section below)

### site__mobile-header

In the `SiteHeader` style components that you just made, add an option for a `mobile` prop.  If the prop is set to `mobile`, then do not output `flex-shrink: 0`. 

## Use Styled Components

Please update the `Layout.tsx` file to use the new Styled Components.  For example:

```
<div className={classes}> --> <Site>
<div className="site__container"> --> <SiteContainer>
<header className="site__mobile-header"> --> <SiteHeader mobile>
...
```

# Resources

* _site.scss file:  https://github.com/thecodeclimber/bluejay-redparts/blob/main/src/scss/common/_site.scss
* Site.js file https://github.com/thecodeclimber/bluejay-redparts/blob/main/src/styled-components/common/Site.js
* Layout.tsx: https://github.com/thecodeclimber/bluejay-redparts/blob/main/src/components/Layout.tsx

# Acceptance Criteria

- [x] Create all Site Styled Components
- [x] Update html tags in `Layouts.tsx`  with new Styled Components

**Comment by thecodeclimber** [comment]
@lokesh75way 

please get started on this (while also finishing up #16.  Thanks.

**Comment by lokesh75way** [comment]
Hi  @thecodeclimber  

This branch **(`Create & Implement "Site" Styled Components #5`)** depends on 
branch **(`issue/#2--setup-styled-components-theming`)** ?  Do I create this branch from **`develop`** branch or from **`issue/#2--setup-styled-components-theming`** branch?

**Comment by thecodeclimber** [comment]
I merged issue/#2—setup-styled-components-theming

**Comment by lokesh75way** [comment]
Github commit link to verify code: https://github.com/thecodeclimber/bluejay-redparts/commit/9e749e324a739c5be528eaf87ebb10008fecab62

PR link: https://github.com/thecodeclimber/bluejay-redparts/pull/18

@thecodeclimber
